source/wp-content/plugins/wporg-release-version/wporg-release-version.php

```diff
@@ -0,0 +1,65 @@
+<?php
+/**
+ * Plugin Name:       Wporg Release Version Block
+ * Description:       Displays the WP release version for a News post.
+ * Author:            WordPress.org
+ * Author URI:        https://wordpress.org/
+ * License:           GPL v2 or later
+ * License URI:       https://www.gnu.org/licenses/gpl-2.0.html
+ * Text Domain:       wporg
+ */
+
+/**
+ * Server-side rendering of the `wporg/release-version` block.
+ */
+
+/**
+ * Renders the `wporg/release-version` block on the server.
+ *
+ * @param array    $attributes Block attributes.
+ * @param string   $content    Block default content.
+ * @param WP_Block $block      Block instance.
+ *
+ * @return string Returns the release version for the current post wrapped inside "h1" tags.
+ */
+function render_block_wporg_release_version( $attributes, $content, $block ) {
+	if ( ! isset( $block->context['postId'] ) ) {
+		return '';
+	}
+
+	$post_ID          = $block->context['postId'];
+	$tag_name         = 'h2';
+	$align_class_name = empty( $attributes['textAlign'] ) ? '' : "has-text-align-{$attributes['textAlign']}";
+
+	if ( isset( $attributes['level'] ) ) {
+		$tag_name = 0 === $attributes['level'] ? 'p' : 'h' . $attributes['level'];
+	}
+
+	$version = '';
+	$title = get_the_title( $post_ID );
+	// Do we also want x.y.z?
+	if ( preg_match( '/WordPress (\d+[.]\d+)/', $title, $matches ) ) {
```

Comment:
A regex that will work for getting 3rd-level versions is `(\d{0,3}(\.\d{1,3})+)` (from [here](https://github.com/WordPress/wporg-news-2021/issues/38#issuecomment-931804064))

Reply:
👍🏻  It seems important to get `.z` in there